panic: descriptor ... is invalid: "aws_rds_ebsbyte_balance%_average" is not a valid metric name

Using container image "quay.io/invisionag/yet-another-cloudwatch-exporter:v0.25.0-alpha"

Here is how I can crash YACE :
```
$ aws cloudwatch list-metrics --namespace "AWS/RDS"
...
        },
        {
            "Namespace": "AWS/RDS",
            "MetricName": "EBSByteBalance%",
            "Dimensions": []
        }
    ]
}
```

Let's configure YACE to retrieve metric `EBSByteBalance%` :
```
---
discovery:
  jobs:
  - type: rds 
    regions:
      - eu-west-3
    metrics:
      - name: 'EBSByteBalance%'
        statistics:
          - Average
        period: 300 
        length: 300 
        delay: 300 
        addCloudwatchTimestamp: true
```

Now run YACE with this configuration file. It will crash soon : 
```
...
panic: descriptor Desc{fqName: "aws_rds_ebsbyte_balance%_average", help: "Help is not implemented yet.", constLabels: {}, variableLabels: []} is invalid: "aws_rds_ebsbyte_balance%_average" is not a valid metric name

goroutine 17 [running]:
github.com/prometheus/client_golang/prometheus.(*Registry).MustRegister(0xc000380000, 0xc00008de68, 0x1, 0x1)
	/go/pkg/mod/github.com/prometheus/client_golang@v1.9.0/prometheus/registry.go:403 +0xad
main.updateMetrics(0xc000380000, 0x0, 0x0, 0x0, 0x405835, 0xc0000690e0, 0xc000048780)
	/opt/main.go:94 +0x33b
main.main.func1(0xc0000a3200, 0xc0000a4190, 0xc0000b75c8)
	/opt/main.go:150 +0xfa
created by main.main
	/opt/main.go:146 +0x63a
```

I guess the fix may be to eliminate or escape special chars like `%` in metric names.

--- a/pkg/prometheus.go
+++ b/pkg/prometheus.go
@@ -126,17 +126,17 @@ func combineLabels(labels map[string]string) string {
 
 func promString(text string) string {
 	text = splitString(text)
-	return strings.ToLower(replaceWithUnderscores(text))
+	return strings.ToLower(sanitize(text))
 }
 
 func promStringTag(text string, labelsSnakeCase bool) string {
 	if labelsSnakeCase {
 		return promString(text)
 	}
-	return replaceWithUnderscores(text)
+	return sanitize(text)
 }
 
-func replaceWithUnderscores(text string) string {
+func sanitize(text string) string {
 	replacer := strings.NewReplacer(
 		" ", "_",
 		",", "_",

--- a/pkg/prometheus.go
+++ b/pkg/prometheus.go
@@ -151,6 +151,7 @@ func replaceWithUnderscores(text string) string {
 		"@", "_",
 		"<", "_",
 		">", "_",
+		"%", "_percent",
 	)
 	return replacer.Replace(text)
 }

